batch/test/test_accounts.py

```diff
@@ -279,6 +279,16 @@ async def test_add_and_delete_user(dev_client: BatchClient, new_billing_project:
     assert r['user'] not in bp['users']
 
 
+async def test_error_adding_nonexistent_user(dev_client: BatchClient, new_billing_project: str):
+    project = new_billing_project
+    try:
+        await dev_client.add_user('foobar', project)
```

Comment:
I know this is very unlikely, but I think for clarity it's worth adding an assert before this line where you `get_user` and assert that it doesn't exist.